lib/CL/clSetKernelArg.c

```diff
@@ -155,7 +155,35 @@ pocl_verify_dbk_kernel_args (cl_mem buf,
     case POCL_CDBI_DBK_EXP_JPEG_ENCODE:
```

Comment:
Can we move the DBK handling to a DBK utility module? Otherwise it will bloat the clSetKernelArg.c.

Reply:
That could be done after this pr, it might be easy to move to another existing file.